### PR TITLE
Replace Xdebug by Blackfire as the default PHP image

### PR DESCRIPTION
**Pull request by ajardin**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -16,5 +16,5 @@ MYSQL_PASSWORD=
 MYSQL_DATABASE=
 
 ##### PHP
-PHP_IDE_CONFIG=serverName=
-XDEBUG_CONFIG=remote_host= idekey=PHPSTORM remote_port=9001 remote_enable=1 remote_connect_back=0
+PHP_IDE_CONFIG=
+XDEBUG_CONFIG=
```
Comment (Fulmenef) [inline .env.dist]: Why don't you keep the configuration, so we can easily switch if needed ? :)

Comment (maxhelias) [inline .env.dist]: You must be comment these lines

Comment (ajardin) [reply .env.dist]: Because there is a dedicated page in the Wiki.

Comment (Fulmenef) [reply .env.dist]: Fair point.

**Pull request merged**